_posts/2018-03-20-infra.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Tech at HackGT: Infrastructure"		
+layout: posts		
+categories: blog posts		
+author: Ehsan Asdar
+image: infrat.png
+---
+[HackGT](https://hack.gt) is a student organization at Georgia Tech that runs a series of events every year that empower students to learn and hone their skills in Computer Science. Our marquee event (of the same name, HackGT) takes place each fall, hosting over 1000 hackers at Georgia Tech for 36 hours of learning and creating, while other events such as [Catalyst](http://catalyst.hack.gt) and [HackGTeeny](http://teeny.hack.gt) strive to show students how intriguing and rewarding careers in STEM, and specifically CS, can be.
+{% render_image  infra1.gif%}
+
+The HackGT tech team builds software to support our event organizing teams. Our variety of responsibilities range from developing educational content (workshops, tech talks, CTFs) to writing applications to manage hacker data. This is the first in a series of posts about HackGT and will focus on how we built our infrastructure to ensure a clean, consistent development-release cycle, complete with zero-downtime deployment, monitoring and alerting, and effortless metrics collection.
+
+# Motivation
```

Comment:
This should probably be `##` (h2). The blog post title is already `h1` so semantically all the headers inside the post should be one level below that.